README.md

```diff
@@ -7,6 +7,154 @@ Jenkins JIRA Plugin
 
 Check documentation at [JIRA plugin page](https://plugins.jenkins.io/jira).
 
+
+Older versions of this plugin may not be safe to use. Please review the
+following warnings before using an older version:
+
+-   [CSRF vulnerability and missing permission checks allowed capturing credentials](https://jenkins.io/security/advisory/2018-09-25/#SECURITY-1029){.external-link}
+-   [Folder-scoped Jira sites are able to access System-scoped credentials](https://jenkins.io/security/advisory/2019-11-21/#SECURITY-1106){.external-link}
```

Comment:
You should delete these warnings, they are automatically shown on the sidebar of the plugin site (e.g. [script-security](https://plugins.jenkins.io/script-security)).

Reply:
good catch done